service/controller/app/resource/validation/resource.go

```diff
@@ -18,7 +18,8 @@ type Config struct {
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
 
-	Provider string
+	ProjectName string
+	Provider    string
```

Comment:
This wiring is needed to pass for the validation package in the app resource. This is to it keeps the validation of configmap and secret names.